### ISSUE TITLE
Custom element doesn't exist: hui-...

### ISSUE THREAD

**Issue by VDRainer** [home-assistant/frontend#4984]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

I'm seeing this since beta 0.106 from time to time, now i found a way to reproduce it.
![Auswahl_293](https://user-images.githubusercontent.com/26381449/75277515-7094bd00-5808-11ea-9117-f7f67f179a3e.png)

## Steps to reproduce

Create a view and add this entities card
```
type: entities
title: First card
show_header_toggle: false
entities:
  - type: 'custom:hui-gauge-card'
    entity: input_number.slider1
  - type: 'custom:hui-gauge-card'
    entity: input_number.slider1
```
The red cards show up for every gauge card.
This could be fixed by: (and that's the reason why i've seen this only from time to time)
- Opening another view that contains a gauge card and going back to the view with the entities card
- Creating a normal gauge card in the view, also if it's not visible
```
type: conditional
conditions:
  - entity: light.bed_light
    state: 'on'
card:
  type: gauge
  min: 0
  max: 100
  theme: default
  entity: input_number.slider1
```

Another example with a markdown and the state-switch by @thomasloven :
```
type: vertical-stack
cards:
  - type: entities
    entities:
      - switch.test_switch
  - type: 'custom:state-switch'
    entity: switch.test_switch
    states:
      'on':
        type: markdown
        content: |
          switch is ON
      'off':
        type: markdown
        content: |
          switch is OFF
```
After reloading the card shows like this
![Auswahl_294](https://user-images.githubusercontent.com/26381449/75278260-d9306980-5809-11ea-90c3-42871e163dab.png)
Going to another tab with a markdown in it and back again, the card shows the expected markdown.
![Auswahl_295](https://user-images.githubusercontent.com/26381449/75278387-11d04300-580a-11ea-9bcf-62f7d5bf1bea.png)

## Environment

- Home Assistant release with the issue: 0.106.0bx
- Last working Home Assistant release (if known): 0.105.4
- UI Type (States or Lovelace): LL
- Browser and browser version: FF and Chrome
- Operating system: Ubuntu

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt
Nothing, absolutely clean.
```

**Comment by arsaboo** [comment]
Custom cards have to be updated to work with .106. Not all custom cards have been updated

**Comment by VDRainer** [comment]
Thanks @arsaboo , but the first example has nothing to do with a custom card, only the old custom:hui-.. trick.

**Comment by iantrich** [comment]
You're right, and is due to the lazy-loading being added to help load only necessary resources.
Unfortunately, the trick to use other elements was an oversight and not a feature. I'll leave it up to @balloob and @bramkragten on where we stand with this. This will likely impact quite a lot of configurations.

**Comment by balloob** [comment]
This is not a supported feature and its current form is not something that we can support either.

Open for discussion on how to solve this. 

For now I suggest you make a resource that preloads specific card types:

```ts
const preloadCard = type => window.loadCardHelpers()
  .then(({ createCardElement }) => createCardElement({type}))

preloadCard("markdown");
preloadCard("gauge");
```

**Comment by VDRainer** [comment]
Closing this for two weeks? :grin:

**Comment by Fabenissimo** [comment]
Looks like HA cast is affected too, I didn't have any luck with creating a custom resource file which preloads all my cards/rows or this https://github.com/gadgetchnnel/lovelace-card-preloader for HA Cast.

Wouldn't it be possible to make a setting in the UI/config whether to use lazy-loading or not? I can't use my Nest Hub anymore which is really a shame.

**Comment by thomasloven** [comment]
The `custom:hui-` trick was a dirty hack from the start, and worked by accident rather than design.
It was never meant to work, but luckily it did. Support will not be reintroduced.

Consider this the closes you'll ever get to an "official" fix:
https://github.com/thomasloven/lovelace-hui-element

**Comment by tinuva** [comment]
I am a bit confused that this happened. I just used the lovelace gui editor to add a picture-glance card and now it is broken on the HA Android App, but it works on the web interface on PC.

![Screen Shot 2020-05-19 at 11 38 34 AM](https://user-images.githubusercontent.com/5417682/82310922-4d237d80-99c5-11ea-9f8e-4fb89a94f9b5.png)

So this is now what I see in the android app:
![f03be3cb-1f55-44cb-ab10-596757da38d6](https://user-images.githubusercontent.com/5417682/82310947-557bb880-99c5-11ea-8418-d49de7fc27ce.jpeg)

So I read through the original issue #5320 and noticed it is linked to this issue here.

Read through everything.

Even tried the workaround posted by @thomasloven 

![fa3a9cc7-b9ff-4394-9d1c-439f63eda407](https://user-images.githubusercontent.com/5417682/82311009-6debd300-99c5-11ea-9584-8c9e0719310b.jpeg)

Again, this works on the web interface, but as seen not in the Android App.

Is there a different place we should log this issue?

What are my options for looking at a camera feed?

**Comment by tinuva** [comment]
Maybe my issue is I tried this:

```
camera_image: camera.front_camera
camera_view: live
card_type: picture-glance
entities:
  - entity: camera.front_camera
label: Important things
row_type: section
title: Front Camera
type: 'custom:hui-element'
```

What am I suppposed to use in that hui-element card from @thomasloven

**Comment by bramkragten** [comment]
Don't respond on closed issues. 
Besides, your issue doesn't seem related to this issue. (you didn't use `custom:` right?) Your issue should be solved in 0.110.
